package.json

```diff
@@ -37,8 +37,8 @@
   },
   "lint-staged": {
     "linters": {
-      "**/*.{ts,tsx,js,jsx,json,html,css,yaml}": [
-        "prettier --write",
+      "**/*.{ts,tsx,js,jsx,json,css,html,yaml}": [
+        "prettier --config ./.prettierrc --ignore-path ./.prettierignore --write '**/*.{ts,tsx,js,jsx,json,html,css,yaml}'",
```

Comment:
I think you dom't have to explicitly set config and ignore-path flags if .prettierrc and .prettierignore are on root

and if you do `--write '**/*.{ts,tsx,js,jsx,json,html,css,yaml}'` prettier is invoked with all files with those extensions, and not only those provided by lint-staged/husky